### ISSUE TITLE
EDA and feature coding

### ISSUE THREAD

**Issue by 4agatha** [4agatha/supervised-learning#1]
We recommend encoding features using OneHotEncoder from sklearn.preprocessing. This tool allows you to train an encoder on a training set and apply the trained entity to the validation and test set
